src/system/NewForm/FormAutocompleteMultiselect.stories.jsx

```diff
@@ -0,0 +1,107 @@
+/** @jsxImportSource theme-ui */
+
+/**
+ * Internal dependencies
+ */
+import { useState } from 'react';
+import * as Form from '.';
+
+export default {
+	title: 'Form/AutocompleteMulti',
+	argTypes: {
+		placeholder: {
+			type: { name: 'string', required: false },
+			control: { type: 'text' },
+		},
+		label: {
+			type: { name: 'string', required: false },
+			control: { type: 'text' },
+		},
+	},
+};
+
+const options = [
+	{ value: 'chocolate', label: 'Chocolate' },
+	{ value: 'strawberry', label: 'Strawberry Chocolate Vanilla Chocolate Vanilla' },
+	{ value: 'vanilla', label: 'Vanilla' },
+	{ value: 'pistachio', label: 'Pistachio' },
+	{ value: 'bubblegum', label: 'Bubblegum' },
+	{ value: 'ube', label: 'Ube' },
+	{ value: 'mango', label: 'Mango' },
+	{ value: 'buko', label: 'Buko' },
+	{ value: 'durian', label: 'Durian' },
+	{ value: 'lecheflan', label: 'Leche Flan' },
+	{
+		value: 'averylongicecreamflavor',
+		label: 'This is a very long ice cream flavorrrr with no name but it tastes goooooooood...',
```

Comment:
@kat3samsin if this will be used for domains we should also add an example with no spaces. Spaces might help with wrapping, depending on how the `whitespace` CSS property is configured, whereas with no space we hardly have wrapping.

Reply:
Thanks for the suggestion! I'll create an example with long domain names.